docker/mkimage

```diff
@@ -15,6 +15,18 @@ readonly DOCKER_UID="${UID:-0}"
 readonly DOCKER_GID="$(id -g)"
 readonly DOCKER_USER="${USER:-root}"
 
+if [ -v LLVM11 ]; then
```

Comment:
I think it is discouraged only because old versions of bash do not support -v.
Since this is part of the docker build process, I'm happy to assume that people have an up to date machine.